tutor/commands/plugins.py

```diff
@@ -153,22 +153,20 @@ def enable(context: Context, plugin_names: list[str]) -> None:
 @click.argument(
     "plugin_names", metavar="plugin", nargs=-1, type=PluginName(allow_all=True)
 )
-@click.pass_obj
-def disable(context: Context, plugin_names: list[str]) -> None:
-    config = tutor_config.load_minimal(context.root)
+@click.pass_context
```

Comment:
Unless I'm mistaken, we no longer have to pass the context here, right? Same for the `enable` command above.

Reply:
yes.